### PR TITLE
New tsvs walls

### PR DESCRIPTION
**Pull request by mpathak24**
adds wall type; cathedral ceiling probabilities as a characteristics. adds as dependency to related insulation tsvs; mocked up the file for Insulation wall for review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -62,4 +62,11 @@ node_modules/
 /workflows/out.osw
 /workflows/results
 
-/resources/buildstock.csv
\ No newline at end of file
+/resources/buildstock.csv
+/_project_resstock_comed
```
Comment (joseph-robertson) [inline .gitignore]: These are pretty messy.

**Comment by joseph-robertson** [comment]
I also don't see anything related to cathedral ceilings in this PR.

**Comment by mpathak24** [comment]
> Seems like you'll want to introduce the "Geometry Wall Type.tsv" file with dummy options into the testing project, and actually run the project to make sure everything is working correctly.
> 
> You also need to update outputs in PAT for each project that gets this new "Geometry Wall Type.tsv" file. That way `building_characteristics_report.geometry_wall_type` will show up in the results csv.

sounds good. will do.

**Comment by ejhw** [comment]
> I also don't see anything related to cathedral ceilings in this PR.

@mpathak24 I'd say proceed with making these requested changes and then make a separate PR for ceilings.

File: project_singlefamilydetached/housing_characteristics/Insulation Wall.tsv [hunk 2]
```diff
@@ -1,71 +1,141 @@
...
+CR02	2000s	CMU wall	0	0	0	0	0	1
+CR05	2000s	CMU wall	0	0	0	0	0	1
+CR03	2000s	CMU wall	0	0	0	0	0	1
+CR04	2000s	CMU wall	0	0	0	0	0	1
+CR06	2000s	CMU wall	0	0	0	0	0	1
+CR07	2000s	CMU wall	0	0	0	0	0	1
+CR08	2000s	CMU wall	0	0	0	0	0	1
+CR09	2000s	CMU wall	0	0	0	0	0	1
+CR10	2000s	CMU wall	0	0	0	0	0	1
+CR11	2000s	CMU wall	0	0	0	0	0	1
```
[the remark this answers is not in the document]
Comment (ejhw) [reply project_singlefamilydetached/housing_characteristics/Insulation Wall.tsv]: @mpathak24 will review any progress and touch base on any questions about how to move forward.

File: project_singlefamilydetached/housing_characteristics/Insulation Wall.tsv [hunk 3]
```diff
@@ -1,71 +1,141 @@
...
-CR05	2000s	0	0	0.14	0.14	0.71
-CR03	2000s	0	0	0.2	0.05	0.75
-CR04	2000s	0	0	0.37	0.32	0.31
-CR06	2000s	0	0	0.09	0	0.91
-CR07	2000s	0	0	0.39	0.03	0.58
-CR08	2000s	0	0	0.6	0.22	0.18
-CR09	2000s	0	0	0.67	0.15	0.17
-CR10	2000s	0	0	0.41	0.2	0.39
-CR11	2000s	0	0	0.47	0.02	0.51
+Dependency=Location Region	Dependency=Vintage	Dependency=Geometry Wall Type	"Option=Wood Stud, Uninsulated"	"Option=Wood Stud, R-7"	"Option=Wood Stud, R-11"	"Option=Wood Stud, R-15"	"Option=Wood Stud, R-19"	"Option=CMU, 6-in Concrete Filled"
```
[the remark this answers is not in the document]
Comment (ejhw) [reply project_singlefamilydetached/housing_characteristics/Insulation Wall.tsv]: @mpathak24 will review any progress and touch base on any questions about how to move forward.

File: project_singlefamilydetached/housing_characteristics/Insulation Wall.tsv (hunk 2 above)
[the remark this answers is not in the document]
Comment (mpathak24) [reply project_singlefamilydetached/housing_characteristics/Insulation Wall.tsv]: Same insulation levels as Wood added.

**Pull request closed without merging**